src/helpers.js

```diff
@@ -22,6 +22,16 @@ const getDirFromFilePath = (filePath: FilePath): FilePath => path.parse(filePath
 const getNearestEslintignorePath = (filePath: FilePath): ?FilePath =>
   findCached(getDirFromFilePath(filePath), '.eslintignore');
 
+const getLocalPrettierPath = (filePath: ?FilePath): ?FilePath => {
+  if (!filePath) return null;
+
+  const indexPath = path.join('node_modules', 'prettier', 'index.js');
+  const dirPath = getDirFromFilePath(filePath);
+  if (!dirPath) return null;
```

Comment:
Is this guard clause possible to hit? If not, you can go FP and avoid state:

```js
const getLocalPrettierPath = (filePath: ?FilePath): ?FilePath => 
  filePath ?
    findCached(getDirFromFilePath(filePath), path.join('node_modules', 'prettier', 'index.js'))
    : null;
```

Also, see my comment in `executePrettier.js`. There's really no reason to not just give the `prettier` instance itself instead of making `executePrettier` do it.